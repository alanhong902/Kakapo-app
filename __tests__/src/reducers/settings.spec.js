import { prop } from 'ramda';
import configureStore from 'stores/configureStore';
import { settingActions } from 'actions/';

const store = configureStore();

const currState = () => prop('settings', store.getState());

test('[reducer/settings]', () => {
  store.dispatch(settingActions.update());
  expect(currState()).toMatchSnapshot();

  store.dispatch(settingActions.update('checking'));
  expect(currState()).toMatchSnapshot();

  store.dispatch(settingActions.mute());
  expect(currState()).toMatchSnapshot();

  store.dispatch(settingActions.mute());
  expect(currState()).toMatchSnapshot();

  store.dispatch(settingActions.dock(true));
  expect(currState()).toMatchSnapshot();

  store.dispatch(settingActions.dock(false));
  expect(currState()).toMatchSnapshot();

  store.dispatch(settingActions.devtools(true));
  expect(currState()).toMatchSnapshot();

  store.dispatch(settingActions.devtools(false));
  expect(currState()).toMatchSnapshot();
});
