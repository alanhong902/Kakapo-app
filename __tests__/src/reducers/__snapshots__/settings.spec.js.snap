// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[reducer/settings] 1`] = `
Object {
  "intlData": Object {
    "id": "en",
    "locales": Array [
      "en",
    ],
    "messages": Object {
      "import.custom.header": "Add a custom stream",
      "import.custom.name_placeholder": "Choose a name for this sound",
      "import.custom.url_placeholder": "Enter a link to an audio file",
      "import.error.empty": "Please fill out all fields",
      "import.error.url": "Please enter a valid url",
      "import.kakapo.header": "Add a sound from Kakapo.com",
      "import.kakapo.subheader": "Below are a collection of sounds curated by us.",
      "import.options.custom": "Custom link",
      "import.options.dragndrop": "You can drag and drop audio files into Kakapo from your computer.",
      "import.options.header": "Add a sound",
      "import.options.kakapo": "Kakapo favourites",
      "import.options.soundcloud": "SoundCloud",
      "import.options.youtube": "YouTube",
      "import.save": "Save",
      "import.soundcloud.header": "Add a sound from SoundCloud",
      "import.soundcloud.search_placeholder": "Search Soundcloud",
      "import.youtube.header": "Add a video from YouTube",
      "import.youtube.search_placeholder": "Search YouTube (or enter a YouTube Link)",
      "list.cancel": "Cancel",
      "list.editing_name": "Name",
      "list.editing_name_placeholder": "Choose a name for this sound",
      "list.editing_tag": "Tag",
      "list.editing_tag_placeholder": "Useful for searching",
      "list.save": "Save",
      "modal.close": "Close",
      "nav.app": "Download Kakapo for iOS / Android, Desktop & Chrome!",
      "nav.downloads": "Downloads",
      "nav.home": "Home",
      "nav.playlist": "Playlists",
      "nav.settings": "Settings",
      "playlist.header": "Playlists",
      "playlist.input_playlist": "Enter a playlist ID below if you have one",
      "playlist.list_camping": "Camping",
      "playlist.list_cosy": "Cosy",
      "playlist.list_night-highway": "Nightime highway",
      "playlist.list_refuge-in-paris-cafe": "Refuge in Paris cafe",
      "playlist.list_reset": "Reset to default sounds",
      "playlist.list_stormy": "Stormy",
      "playlist.list_white-noise": "White noise",
      "playlist.share": "Share my current playlist",
      "playlist.share_created": "Copy and share the link for your playlist below",
      "playlist.subheading": "You can also choose some playlists from the selection below:",
      "settings.header": "Settings",
      "settings.lang": "Language",
      "settings.theme": "Theme colors",
      "soundcloud.plays": "plays",
      "sounds.birds": "Birds",
      "sounds.bonfire": "Bonfire",
      "sounds.cat_purring": "Cat Purring",
      "sounds.chimes": "Chimes",
      "sounds.fan": "Fan",
      "sounds.leaves": "Leaves",
      "sounds.paris_cafe": "Paris Cafe",
      "sounds.paris_street_corner": "Paris Street Corner",
      "sounds.rain": "Rain",
      "sounds.stream": "Stream",
      "sounds.thunder": "Thunder",
      "sounds.train": "Train",
      "sounds.waves": "Waves",
      "sounds.wind": "Wind",
      "youtube.views": "views",
    },
  },
  "lang": "en",
  "mute": null,
  "updateStatus": undefined,
}
`;

exports[`[reducer/settings] 2`] = `
Object {
  "intlData": Object {
    "id": "en",
    "locales": Array [
      "en",
    ],
    "messages": Object {
      "import.custom.header": "Add a custom stream",
      "import.custom.name_placeholder": "Choose a name for this sound",
      "import.custom.url_placeholder": "Enter a link to an audio file",
      "import.error.empty": "Please fill out all fields",
      "import.error.url": "Please enter a valid url",
      "import.kakapo.header": "Add a sound from Kakapo.com",
      "import.kakapo.subheader": "Below are a collection of sounds curated by us.",
      "import.options.custom": "Custom link",
      "import.options.dragndrop": "You can drag and drop audio files into Kakapo from your computer.",
      "import.options.header": "Add a sound",
      "import.options.kakapo": "Kakapo favourites",
      "import.options.soundcloud": "SoundCloud",
      "import.options.youtube": "YouTube",
      "import.save": "Save",
      "import.soundcloud.header": "Add a sound from SoundCloud",
      "import.soundcloud.search_placeholder": "Search Soundcloud",
      "import.youtube.header": "Add a video from YouTube",
      "import.youtube.search_placeholder": "Search YouTube (or enter a YouTube Link)",
      "list.cancel": "Cancel",
      "list.editing_name": "Name",
      "list.editing_name_placeholder": "Choose a name for this sound",
      "list.editing_tag": "Tag",
      "list.editing_tag_placeholder": "Useful for searching",
      "list.save": "Save",
      "modal.close": "Close",
      "nav.app": "Download Kakapo for iOS / Android, Desktop & Chrome!",
      "nav.downloads": "Downloads",
      "nav.home": "Home",
      "nav.playlist": "Playlists",
      "nav.settings": "Settings",
      "playlist.header": "Playlists",
      "playlist.input_playlist": "Enter a playlist ID below if you have one",
      "playlist.list_camping": "Camping",
      "playlist.list_cosy": "Cosy",
      "playlist.list_night-highway": "Nightime highway",
      "playlist.list_refuge-in-paris-cafe": "Refuge in Paris cafe",
      "playlist.list_reset": "Reset to default sounds",
      "playlist.list_stormy": "Stormy",
      "playlist.list_white-noise": "White noise",
      "playlist.share": "Share my current playlist",
      "playlist.share_created": "Copy and share the link for your playlist below",
      "playlist.subheading": "You can also choose some playlists from the selection below:",
      "settings.header": "Settings",
      "settings.lang": "Language",
      "settings.theme": "Theme colors",
      "soundcloud.plays": "plays",
      "sounds.birds": "Birds",
      "sounds.bonfire": "Bonfire",
      "sounds.cat_purring": "Cat Purring",
      "sounds.chimes": "Chimes",
      "sounds.fan": "Fan",
      "sounds.leaves": "Leaves",
      "sounds.paris_cafe": "Paris Cafe",
      "sounds.paris_street_corner": "Paris Street Corner",
      "sounds.rain": "Rain",
      "sounds.stream": "Stream",
      "sounds.thunder": "Thunder",
      "sounds.train": "Train",
      "sounds.waves": "Waves",
      "sounds.wind": "Wind",
      "youtube.views": "views",
    },
  },
  "lang": "en",
  "mute": null,
  "updateStatus": "checking",
}
`;

exports[`[reducer/settings] 3`] = `
Object {
  "intlData": Object {
    "id": "en",
    "locales": Array [
      "en",
    ],
    "messages": Object {
      "import.custom.header": "Add a custom stream",
      "import.custom.name_placeholder": "Choose a name for this sound",
      "import.custom.url_placeholder": "Enter a link to an audio file",
      "import.error.empty": "Please fill out all fields",
      "import.error.url": "Please enter a valid url",
      "import.kakapo.header": "Add a sound from Kakapo.com",
      "import.kakapo.subheader": "Below are a collection of sounds curated by us.",
      "import.options.custom": "Custom link",
      "import.options.dragndrop": "You can drag and drop audio files into Kakapo from your computer.",
      "import.options.header": "Add a sound",
      "import.options.kakapo": "Kakapo favourites",
      "import.options.soundcloud": "SoundCloud",
      "import.options.youtube": "YouTube",
      "import.save": "Save",
      "import.soundcloud.header": "Add a sound from SoundCloud",
      "import.soundcloud.search_placeholder": "Search Soundcloud",
      "import.youtube.header": "Add a video from YouTube",
      "import.youtube.search_placeholder": "Search YouTube (or enter a YouTube Link)",
      "list.cancel": "Cancel",
      "list.editing_name": "Name",
      "list.editing_name_placeholder": "Choose a name for this sound",
      "list.editing_tag": "Tag",
      "list.editing_tag_placeholder": "Useful for searching",
      "list.save": "Save",
      "modal.close": "Close",
      "nav.app": "Download Kakapo for iOS / Android, Desktop & Chrome!",
      "nav.downloads": "Downloads",
      "nav.home": "Home",
      "nav.playlist": "Playlists",
      "nav.settings": "Settings",
      "playlist.header": "Playlists",
      "playlist.input_playlist": "Enter a playlist ID below if you have one",
      "playlist.list_camping": "Camping",
      "playlist.list_cosy": "Cosy",
      "playlist.list_night-highway": "Nightime highway",
      "playlist.list_refuge-in-paris-cafe": "Refuge in Paris cafe",
      "playlist.list_reset": "Reset to default sounds",
      "playlist.list_stormy": "Stormy",
      "playlist.list_white-noise": "White noise",
      "playlist.share": "Share my current playlist",
      "playlist.share_created": "Copy and share the link for your playlist below",
      "playlist.subheading": "You can also choose some playlists from the selection below:",
      "settings.header": "Settings",
      "settings.lang": "Language",
      "settings.theme": "Theme colors",
      "soundcloud.plays": "plays",
      "sounds.birds": "Birds",
      "sounds.bonfire": "Bonfire",
      "sounds.cat_purring": "Cat Purring",
      "sounds.chimes": "Chimes",
      "sounds.fan": "Fan",
      "sounds.leaves": "Leaves",
      "sounds.paris_cafe": "Paris Cafe",
      "sounds.paris_street_corner": "Paris Street Corner",
      "sounds.rain": "Rain",
      "sounds.stream": "Stream",
      "sounds.thunder": "Thunder",
      "sounds.train": "Train",
      "sounds.waves": "Waves",
      "sounds.wind": "Wind",
      "youtube.views": "views",
    },
  },
  "lang": "en",
  "mute": true,
  "updateStatus": "checking",
}
`;

exports[`[reducer/settings] 4`] = `
Object {
  "intlData": Object {
    "id": "en",
    "locales": Array [
      "en",
    ],
    "messages": Object {
      "import.custom.header": "Add a custom stream",
      "import.custom.name_placeholder": "Choose a name for this sound",
      "import.custom.url_placeholder": "Enter a link to an audio file",
      "import.error.empty": "Please fill out all fields",
      "import.error.url": "Please enter a valid url",
      "import.kakapo.header": "Add a sound from Kakapo.com",
      "import.kakapo.subheader": "Below are a collection of sounds curated by us.",
      "import.options.custom": "Custom link",
      "import.options.dragndrop": "You can drag and drop audio files into Kakapo from your computer.",
      "import.options.header": "Add a sound",
      "import.options.kakapo": "Kakapo favourites",
      "import.options.soundcloud": "SoundCloud",
      "import.options.youtube": "YouTube",
      "import.save": "Save",
      "import.soundcloud.header": "Add a sound from SoundCloud",
      "import.soundcloud.search_placeholder": "Search Soundcloud",
      "import.youtube.header": "Add a video from YouTube",
      "import.youtube.search_placeholder": "Search YouTube (or enter a YouTube Link)",
      "list.cancel": "Cancel",
      "list.editing_name": "Name",
      "list.editing_name_placeholder": "Choose a name for this sound",
      "list.editing_tag": "Tag",
      "list.editing_tag_placeholder": "Useful for searching",
      "list.save": "Save",
      "modal.close": "Close",
      "nav.app": "Download Kakapo for iOS / Android, Desktop & Chrome!",
      "nav.downloads": "Downloads",
      "nav.home": "Home",
      "nav.playlist": "Playlists",
      "nav.settings": "Settings",
      "playlist.header": "Playlists",
      "playlist.input_playlist": "Enter a playlist ID below if you have one",
      "playlist.list_camping": "Camping",
      "playlist.list_cosy": "Cosy",
      "playlist.list_night-highway": "Nightime highway",
      "playlist.list_refuge-in-paris-cafe": "Refuge in Paris cafe",
      "playlist.list_reset": "Reset to default sounds",
      "playlist.list_stormy": "Stormy",
      "playlist.list_white-noise": "White noise",
      "playlist.share": "Share my current playlist",
      "playlist.share_created": "Copy and share the link for your playlist below",
      "playlist.subheading": "You can also choose some playlists from the selection below:",
      "settings.header": "Settings",
      "settings.lang": "Language",
      "settings.theme": "Theme colors",
      "soundcloud.plays": "plays",
      "sounds.birds": "Birds",
      "sounds.bonfire": "Bonfire",
      "sounds.cat_purring": "Cat Purring",
      "sounds.chimes": "Chimes",
      "sounds.fan": "Fan",
      "sounds.leaves": "Leaves",
      "sounds.paris_cafe": "Paris Cafe",
      "sounds.paris_street_corner": "Paris Street Corner",
      "sounds.rain": "Rain",
      "sounds.stream": "Stream",
      "sounds.thunder": "Thunder",
      "sounds.train": "Train",
      "sounds.waves": "Waves",
      "sounds.wind": "Wind",
      "youtube.views": "views",
    },
  },
  "lang": "en",
  "mute": false,
  "updateStatus": "checking",
}
`;

exports[`[reducer/settings] 5`] = `
Object {
  "dockIcon": true,
  "intlData": Object {
    "id": "en",
    "locales": Array [
      "en",
    ],
    "messages": Object {
      "import.custom.header": "Add a custom stream",
      "import.custom.name_placeholder": "Choose a name for this sound",
      "import.custom.url_placeholder": "Enter a link to an audio file",
      "import.error.empty": "Please fill out all fields",
      "import.error.url": "Please enter a valid url",
      "import.kakapo.header": "Add a sound from Kakapo.com",
      "import.kakapo.subheader": "Below are a collection of sounds curated by us.",
      "import.options.custom": "Custom link",
      "import.options.dragndrop": "You can drag and drop audio files into Kakapo from your computer.",
      "import.options.header": "Add a sound",
      "import.options.kakapo": "Kakapo favourites",
      "import.options.soundcloud": "SoundCloud",
      "import.options.youtube": "YouTube",
      "import.save": "Save",
      "import.soundcloud.header": "Add a sound from SoundCloud",
      "import.soundcloud.search_placeholder": "Search Soundcloud",
      "import.youtube.header": "Add a video from YouTube",
      "import.youtube.search_placeholder": "Search YouTube (or enter a YouTube Link)",
      "list.cancel": "Cancel",
      "list.editing_name": "Name",
      "list.editing_name_placeholder": "Choose a name for this sound",
      "list.editing_tag": "Tag",
      "list.editing_tag_placeholder": "Useful for searching",
      "list.save": "Save",
      "modal.close": "Close",
      "nav.app": "Download Kakapo for iOS / Android, Desktop & Chrome!",
      "nav.downloads": "Downloads",
      "nav.home": "Home",
      "nav.playlist": "Playlists",
      "nav.settings": "Settings",
      "playlist.header": "Playlists",
      "playlist.input_playlist": "Enter a playlist ID below if you have one",
      "playlist.list_camping": "Camping",
      "playlist.list_cosy": "Cosy",
      "playlist.list_night-highway": "Nightime highway",
      "playlist.list_refuge-in-paris-cafe": "Refuge in Paris cafe",
      "playlist.list_reset": "Reset to default sounds",
      "playlist.list_stormy": "Stormy",
      "playlist.list_white-noise": "White noise",
      "playlist.share": "Share my current playlist",
      "playlist.share_created": "Copy and share the link for your playlist below",
      "playlist.subheading": "You can also choose some playlists from the selection below:",
      "settings.header": "Settings",
      "settings.lang": "Language",
      "settings.theme": "Theme colors",
      "soundcloud.plays": "plays",
      "sounds.birds": "Birds",
      "sounds.bonfire": "Bonfire",
      "sounds.cat_purring": "Cat Purring",
      "sounds.chimes": "Chimes",
      "sounds.fan": "Fan",
      "sounds.leaves": "Leaves",
      "sounds.paris_cafe": "Paris Cafe",
      "sounds.paris_street_corner": "Paris Street Corner",
      "sounds.rain": "Rain",
      "sounds.stream": "Stream",
      "sounds.thunder": "Thunder",
      "sounds.train": "Train",
      "sounds.waves": "Waves",
      "sounds.wind": "Wind",
      "youtube.views": "views",
    },
  },
  "lang": "en",
  "mute": false,
  "updateStatus": "checking",
}
`;

exports[`[reducer/settings] 6`] = `
Object {
  "dockIcon": false,
  "intlData": Object {
    "id": "en",
    "locales": Array [
      "en",
    ],
    "messages": Object {
      "import.custom.header": "Add a custom stream",
      "import.custom.name_placeholder": "Choose a name for this sound",
      "import.custom.url_placeholder": "Enter a link to an audio file",
      "import.error.empty": "Please fill out all fields",
      "import.error.url": "Please enter a valid url",
      "import.kakapo.header": "Add a sound from Kakapo.com",
      "import.kakapo.subheader": "Below are a collection of sounds curated by us.",
      "import.options.custom": "Custom link",
      "import.options.dragndrop": "You can drag and drop audio files into Kakapo from your computer.",
      "import.options.header": "Add a sound",
      "import.options.kakapo": "Kakapo favourites",
      "import.options.soundcloud": "SoundCloud",
      "import.options.youtube": "YouTube",
      "import.save": "Save",
      "import.soundcloud.header": "Add a sound from SoundCloud",
      "import.soundcloud.search_placeholder": "Search Soundcloud",
      "import.youtube.header": "Add a video from YouTube",
      "import.youtube.search_placeholder": "Search YouTube (or enter a YouTube Link)",
      "list.cancel": "Cancel",
      "list.editing_name": "Name",
      "list.editing_name_placeholder": "Choose a name for this sound",
      "list.editing_tag": "Tag",
      "list.editing_tag_placeholder": "Useful for searching",
      "list.save": "Save",
      "modal.close": "Close",
      "nav.app": "Download Kakapo for iOS / Android, Desktop & Chrome!",
      "nav.downloads": "Downloads",
      "nav.home": "Home",
      "nav.playlist": "Playlists",
      "nav.settings": "Settings",
      "playlist.header": "Playlists",
      "playlist.input_playlist": "Enter a playlist ID below if you have one",
      "playlist.list_camping": "Camping",
      "playlist.list_cosy": "Cosy",
      "playlist.list_night-highway": "Nightime highway",
      "playlist.list_refuge-in-paris-cafe": "Refuge in Paris cafe",
      "playlist.list_reset": "Reset to default sounds",
      "playlist.list_stormy": "Stormy",
      "playlist.list_white-noise": "White noise",
      "playlist.share": "Share my current playlist",
      "playlist.share_created": "Copy and share the link for your playlist below",
      "playlist.subheading": "You can also choose some playlists from the selection below:",
      "settings.header": "Settings",
      "settings.lang": "Language",
      "settings.theme": "Theme colors",
      "soundcloud.plays": "plays",
      "sounds.birds": "Birds",
      "sounds.bonfire": "Bonfire",
      "sounds.cat_purring": "Cat Purring",
      "sounds.chimes": "Chimes",
      "sounds.fan": "Fan",
      "sounds.leaves": "Leaves",
      "sounds.paris_cafe": "Paris Cafe",
      "sounds.paris_street_corner": "Paris Street Corner",
      "sounds.rain": "Rain",
      "sounds.stream": "Stream",
      "sounds.thunder": "Thunder",
      "sounds.train": "Train",
      "sounds.waves": "Waves",
      "sounds.wind": "Wind",
      "youtube.views": "views",
    },
  },
  "lang": "en",
  "mute": false,
  "updateStatus": "checking",
}
`;

exports[`[reducer/settings] 7`] = `
Object {
  "devTools": true,
  "dockIcon": false,
  "intlData": Object {
    "id": "en",
    "locales": Array [
      "en",
    ],
    "messages": Object {
      "import.custom.header": "Add a custom stream",
      "import.custom.name_placeholder": "Choose a name for this sound",
      "import.custom.url_placeholder": "Enter a link to an audio file",
      "import.error.empty": "Please fill out all fields",
      "import.error.url": "Please enter a valid url",
      "import.kakapo.header": "Add a sound from Kakapo.com",
      "import.kakapo.subheader": "Below are a collection of sounds curated by us.",
      "import.options.custom": "Custom link",
      "import.options.dragndrop": "You can drag and drop audio files into Kakapo from your computer.",
      "import.options.header": "Add a sound",
      "import.options.kakapo": "Kakapo favourites",
      "import.options.soundcloud": "SoundCloud",
      "import.options.youtube": "YouTube",
      "import.save": "Save",
      "import.soundcloud.header": "Add a sound from SoundCloud",
      "import.soundcloud.search_placeholder": "Search Soundcloud",
      "import.youtube.header": "Add a video from YouTube",
      "import.youtube.search_placeholder": "Search YouTube (or enter a YouTube Link)",
      "list.cancel": "Cancel",
      "list.editing_name": "Name",
      "list.editing_name_placeholder": "Choose a name for this sound",
      "list.editing_tag": "Tag",
      "list.editing_tag_placeholder": "Useful for searching",
      "list.save": "Save",
      "modal.close": "Close",
      "nav.app": "Download Kakapo for iOS / Android, Desktop & Chrome!",
      "nav.downloads": "Downloads",
      "nav.home": "Home",
      "nav.playlist": "Playlists",
      "nav.settings": "Settings",
      "playlist.header": "Playlists",
      "playlist.input_playlist": "Enter a playlist ID below if you have one",
      "playlist.list_camping": "Camping",
      "playlist.list_cosy": "Cosy",
      "playlist.list_night-highway": "Nightime highway",
      "playlist.list_refuge-in-paris-cafe": "Refuge in Paris cafe",
      "playlist.list_reset": "Reset to default sounds",
      "playlist.list_stormy": "Stormy",
      "playlist.list_white-noise": "White noise",
      "playlist.share": "Share my current playlist",
      "playlist.share_created": "Copy and share the link for your playlist below",
      "playlist.subheading": "You can also choose some playlists from the selection below:",
      "settings.header": "Settings",
      "settings.lang": "Language",
      "settings.theme": "Theme colors",
      "soundcloud.plays": "plays",
      "sounds.birds": "Birds",
      "sounds.bonfire": "Bonfire",
      "sounds.cat_purring": "Cat Purring",
      "sounds.chimes": "Chimes",
      "sounds.fan": "Fan",
      "sounds.leaves": "Leaves",
      "sounds.paris_cafe": "Paris Cafe",
      "sounds.paris_street_corner": "Paris Street Corner",
      "sounds.rain": "Rain",
      "sounds.stream": "Stream",
      "sounds.thunder": "Thunder",
      "sounds.train": "Train",
      "sounds.waves": "Waves",
      "sounds.wind": "Wind",
      "youtube.views": "views",
    },
  },
  "lang": "en",
  "mute": false,
  "updateStatus": "checking",
}
`;

exports[`[reducer/settings] 8`] = `
Object {
  "devTools": false,
  "dockIcon": false,
  "intlData": Object {
    "id": "en",
    "locales": Array [
      "en",
    ],
    "messages": Object {
      "import.custom.header": "Add a custom stream",
      "import.custom.name_placeholder": "Choose a name for this sound",
      "import.custom.url_placeholder": "Enter a link to an audio file",
      "import.error.empty": "Please fill out all fields",
      "import.error.url": "Please enter a valid url",
      "import.kakapo.header": "Add a sound from Kakapo.com",
      "import.kakapo.subheader": "Below are a collection of sounds curated by us.",
      "import.options.custom": "Custom link",
      "import.options.dragndrop": "You can drag and drop audio files into Kakapo from your computer.",
      "import.options.header": "Add a sound",
      "import.options.kakapo": "Kakapo favourites",
      "import.options.soundcloud": "SoundCloud",
      "import.options.youtube": "YouTube",
      "import.save": "Save",
      "import.soundcloud.header": "Add a sound from SoundCloud",
      "import.soundcloud.search_placeholder": "Search Soundcloud",
      "import.youtube.header": "Add a video from YouTube",
      "import.youtube.search_placeholder": "Search YouTube (or enter a YouTube Link)",
      "list.cancel": "Cancel",
      "list.editing_name": "Name",
      "list.editing_name_placeholder": "Choose a name for this sound",
      "list.editing_tag": "Tag",
      "list.editing_tag_placeholder": "Useful for searching",
      "list.save": "Save",
      "modal.close": "Close",
      "nav.app": "Download Kakapo for iOS / Android, Desktop & Chrome!",
      "nav.downloads": "Downloads",
      "nav.home": "Home",
      "nav.playlist": "Playlists",
      "nav.settings": "Settings",
      "playlist.header": "Playlists",
      "playlist.input_playlist": "Enter a playlist ID below if you have one",
      "playlist.list_camping": "Camping",
      "playlist.list_cosy": "Cosy",
      "playlist.list_night-highway": "Nightime highway",
      "playlist.list_refuge-in-paris-cafe": "Refuge in Paris cafe",
      "playlist.list_reset": "Reset to default sounds",
      "playlist.list_stormy": "Stormy",
      "playlist.list_white-noise": "White noise",
      "playlist.share": "Share my current playlist",
      "playlist.share_created": "Copy and share the link for your playlist below",
      "playlist.subheading": "You can also choose some playlists from the selection below:",
      "settings.header": "Settings",
      "settings.lang": "Language",
      "settings.theme": "Theme colors",
      "soundcloud.plays": "plays",
      "sounds.birds": "Birds",
      "sounds.bonfire": "Bonfire",
      "sounds.cat_purring": "Cat Purring",
      "sounds.chimes": "Chimes",
      "sounds.fan": "Fan",
      "sounds.leaves": "Leaves",
      "sounds.paris_cafe": "Paris Cafe",
      "sounds.paris_street_corner": "Paris Street Corner",
      "sounds.rain": "Rain",
      "sounds.stream": "Stream",
      "sounds.thunder": "Thunder",
      "sounds.train": "Train",
      "sounds.waves": "Waves",
      "sounds.wind": "Wind",
      "youtube.views": "views",
    },
  },
  "lang": "en",
  "mute": false,
  "updateStatus": "checking",
}
`;
